AWS/Create_IAM_User_with_policy.ipynb

```diff
@@ -0,0 +1,941 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.        "Password": "\\"test@123\\"",
</pre><p>this wont work, please re-review this runbook and fix it properly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
this is as written in the current RunBook. This was not created by me.